wordle-web/pages/game.vue

```diff
@@ -1,116 +1,153 @@
-        </v-alert>
-      </v-row>
-
+      <!-- Full game time in (h)(h)(:)(m)m:ss-->
       <v-row justify="center">
-        <game-board :wordleGame="wordleGame" />
-      </v-row>
+    <gameboard :wordleGame="wordleGame" />
+      </v-row >
       <v-row justify="center">
-        <keyboard :wordleGame="wordleGame" />
+    <keyboard :wordleGame="wordleGame" />
       </v-row>
-    </v-container>
+    </v-col>
   </v-container>
 </template>
 
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator'
+import { Component, Vue, Prop} from 'vue-property-decorator'
 import { WordsService } from '~/scripts/wordsService'
 import { GameState, WordleGame } from '~/scripts/wordleGame'
-import KeyBoard from '@/components/keyboard.vue'
-import GameBoard from '@/components/game-board.vue'
+import KeyBoard from '~/components/keyboard.vue'
+import GameBoard from '~/components/gameboard.vue'
 import { Word } from '~/scripts/word'
+import {Stopwatch} from '~/scripts/stopwatch'
+import {Player} from '~/scripts/player'
+import axios from 'axios'
 
 @Component({ components: { KeyBoard, GameBoard } })
 export default class Game extends Vue {
+  playerName: string = localStorage.getItem('playerName') || "Guest";
+  stopwatch: Stopwatch = new Stopwatch();
+  unposted: boolean = false; // remains false unless a player is prompted to change their name at the end of the game
+
   word: string = WordsService.getRandomWord()
   wordleGame = new WordleGame(this.word)
+  leaderboardPrompt: boolean = false;
 
-  isLoaded: boolean = false
-
-  mounted() {
-    setTimeout(() => {
-      this.isLoaded = true
-    }, 5000)
+  mounted(){
+    if(!this.stopwatch.isRunning){
+      this.stopwatch.Start();
+    }
   }
 
+
   resetGame() {
     this.word = WordsService.getRandomWord()
     this.wordleGame = new WordleGame(this.word)
+    this.unposted = false // Should never be needed
+    this.stopwatch.Start();
   }
 
   get gameResult() {
     if (this.wordleGame.state === GameState.Won) {
-      return { type: 'success', text: 'You won! :^)' }
+      this.endOfGame();
+      return { type: 'success', text: 'Yay! You won!' }
     }
     if (this.wordleGame.state === GameState.Lost) {
-      return {
-        type: 'error',
-        text: `You lost... :^( The word was ${this.word}`,
-      }
+      this.endOfGame();
+      return { type: 'error', text: `You lost... :( The word was ${this.word}` }
     }
     return { type: '', text: '' }
   }
 
+  endOfGame(){
+    this.stopwatch.Stop();
+      if(this.playerName === "Guest"){
+        this.unposted = true;
+
+        this.leaderboardPrompt = true;
+      }
+  }
+
   getLetter(row: number, index: number) {
     const word: Word = this.wordleGame.words[row - 1]
     if (word !== undefined) {
-      return word.letters[index - 1]?.char ?? ''
+      return word.letters[index - 1]?.char ?? '' 
     }
     return ''
   }
+
+  cancelNameEntry(originalName :string) {
```

Comment:
It doesn't look like this parameter is being used. Be sure to run your `npm run lintfix` before your final submission to make sure you catch things like this!